pkg/manifests/manifests.go

```diff
@@ -109,6 +109,17 @@ func WithNamespace(ns string) func(mf *Manifests) {
 		mf.Role.Namespace = ns
 		mf.RB.Namespace = ns
 		mf.SA.Namespace = ns
+		// SCC is cluster scoped
+	}
+}
+
+func WithName(name string) func(mf *Manifests) {
+	return func(mf *Manifests) {
```

Comment:
but do we _want_ to have the same name for all the objects? what's the rationale?

Reply:
When the client controls the name, it can store the name and delete the objects without having the need to hold the objects themselves.

Reply:
I think this is ok and has no ill effect but now I'm thinking if this is the best way to achieve this goal. IIRC using the `ControllerReference` is the best/preferred approach.
TL;DR: I'm fine with using the common name, but let's make sure address the actual need (make it easy to delete objects in the right order without leftovers)

Reply:
In our case for example we want to be able to disable the feature (delete the objects) without deleting the profile

Reply:
This is a very good point and deserves a clear comment in the codebase. So we need a partial reimplementation of `ControllerReference`. I'm wondering if we can add an annotation, or a `ObjectReference` to get the same functionality. This can totally deferred, but I'd rather look into this sooner or later to look for the best practices according to k8s standards, and I'd check if there are precedents or similar enough cases which can help.
For now I recommend to move on with the common name per your approach and add a note to check for the future.

Reply:
This sounds more and more like it deserves a fully functional controller of it's own :)